ecs-init/backoff/backoff.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package backoff
+
+import (
+	"math"
+	mathrand "math/rand"
```

Comment:
Addressed this and other comments in the new [commit](https://github.com/aws/amazon-ecs-init/pull/78/commits/8dc38d74875365902787c331e09a458271234ce6).